dashboard/src/shared/AxiosInstance.test.ts

```diff
@@ -181,4 +181,22 @@ describe("createAxiosInterceptorWithAuth", () => {
     }
     expect(Auth.unsetAuthCookie).toHaveBeenCalled();
   });
+
+  it("parses a forbidden response", async () => {
```

Comment:
We had a bit of a UX regression since the calls to `kubeops` that return a list of required RBAC roles was not being properly rendered in the error message.